.gitignore

```diff
@@ -19,4 +19,5 @@ tmp
 
 .gems
 .ruby-version
-.rbenv-gemsets
\ No newline at end of file
+.rbenv-gemsets
+.idea/
```

Comment:
You should not commit this in our public repo

Reply:
I reverted this to previous .gitignore version